Decision records/05 - We use Details and List to describe index- and show-style route components.md

```diff
@@ -0,0 +1,43 @@
+# We use `Details` and `List` to describe index- and show-style route components
+
+## Date
+
+July 30, 2018
+
+## Contributors
+
+* Chris Sauvé
+* Mallory Allen
+* Tzvi Melamed
+* Ismail Syed
+* Gord Pearson
+* Matt Seccafien
+
+## Summary
+
+Many apps will have components that represent discrete routes in the application. These routes often represent traditional CRUD-style operations on a resource. In these cases, we use the terms "Details" (for a single resource) and "List" (for a list of resources) to name these routes.
```

Comment:
This also makes it easier to concept what it might look like if we ever need the concept of a details and edit page like mobile has. 👍